src/components/ScroolToTop.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import { Box } from '@mui/material';
+
+const UpDownButton = () => {
+  const [isVisible, setIsVisible] = useState(false);
+  const [isAtBottom, setIsAtBottom] = useState(false);
+
+  const trackScroll = () => {
+    const scrolled = window.pageYOffset;
```

Comment:
@shreyas1434shinde wrap all the window uses inside `    if (typeof window !== 'undefined' && window.localStorage) {`

Reply:
@itsvick resolved